Dockerfile

```diff
@@ -44,9 +44,11 @@ WORKDIR /data
 CMD ["/usr/local/bin/ocrd", "--help"]
 
 FROM ocrd_core_base as ocrd_core_test
+# Optionally skip make assets with this arg
+ARG SKIP_ASSETS
 WORKDIR /build-ocrd
 COPY Makefile .
-RUN make assets
+RUN if test -z "$SKIP_ASSETS" || test $SKIP_ASSETS -eq 0 ; then make assets ; fi
```

Comment:
I would expect `SKIP_ASSETS=0` to disable the behavior.  Did you mean `test $SKIP_ASSETS -eq 1`?

Reply:
I think `SKIP_ASSETS` should be renamed to `MAKE_ASSETS` to reverse the logic.

`SKIP_ASSETS` itself has a negative meaning. False to False makes True.

SKIP_ASSETS=1 (true) -> do not make assets
SKIP_ASSETS=0 (false) -> make assets

Reply:
for me 0 is false so if `SKIP_ASSETS` is 0 then it is not skipped.